test/plugins/collections.test.js

```diff
@@ -167,21 +165,58 @@ describe('collection plugin test', () => {
                     protocol: reply.request.server.info.protocol,
                     pathname: `${options.url}/${testCollection.id}`
                 };
+                const expected = Object.assign({}, testCollection);
 
-                const expected = {
-                    name: testCollection.name,
-                    description: testCollection.description,
-                    pipelineIds: testCollection.pipelineIds,
-                    userId: testCollection.userId
+                delete expected.id;
+                assert.calledWith(collectionFactoryMock.create, expected);
+                assert.equal(reply.statusCode, 201);
+                assert.deepEqual(reply.result, testCollection);
+                assert.strictEqual(reply.headers.location, urlLib.format(expectedLocation));
+            })
+        );
+
+        it('returns 201 and creates a collection when no pipelineIds given', () => {
+            delete options.payload.pipelineIds;
+
+            return server.inject(options).then((reply) => {
+                const expectedLocation = {
+                    host: reply.request.headers.host,
+                    port: reply.request.headers.port,
+                    protocol: reply.request.server.info.protocol,
+                    pathname: `${options.url}/${testCollection.id}`
                 };
+                const expected = Object.assign({}, testCollection);
 
+                delete expected.id;
+                delete expected.pipelineIds;
```

Comment:
It doesn't need it to be passed in to `create` I'm already handling it on the models end and I recently got a [PR](https://github.com/screwdriver-cd/models/pull/184) merged testing it out